docs/getting_started/bundles_starter_pack.md

```diff
@@ -0,0 +1,51 @@
+# Bundles starter pack
+
+eZ Platform application is a collection of bundles, similar to Symfony where ["everything is a bundle"](http://symfony.com/doc/current/book/bundles.html). Bundles listed below are useful additions to your clean eZ installation by using them you can improve your Platform usage.
+
+For more information about eZ bundle structure checkout:
```

Comment:
`eZ Platform's bundle...` (let's avoid using `eZ` by itself unless referring to the company specifically).
`check out` - two words